src/core/api/views.py

```diff
@@ -83,3 +85,63 @@ def submit(self, request, *args, **kwargs):
     @action(methods=['get'], detail=False)
     def approval_current(self, request, *args, **kwargs):
         return self.list(request, args, kwargs)
+
+
+class FeedbackCountView(APIView):
+
+    def get(self, request, format=None):
+        today = date.today()
+        mtech_submitted = Feedback.objects.filter(date_submitted__month=today.month,
+                                                  assignment__teaching_assistant__program="1").count()
+        mtech_approved = Feedback.objects.filter(date_submitted__month=today.month,
+                                                 assignment__teaching_assistant__program="1", status="2").count()
+        mtech_rejected = Feedback.objects.filter(date_submitted__month=today.month,
+                                                 assignment__teaching_assistant__program="1", status="3").count()
+        mtech_pending = Feedback.objects.filter(date_submitted__month=today.month,
+                                                assignment__teaching_assistant__program="1", status="1").count()
+
+        phd_mhrd_submitted = Feedback.objects.filter(date_submitted__month=today.month,
+                                                     assignment__teaching_assistant__program="2").count()
+        phd_mhrd_approved = Feedback.objects.filter(date_submitted__month=today.month,
+                                                    assignment__teaching_assistant__program="2", status="2").count()
+        phd_mhrd_rejected = Feedback.objects.filter(date_submitted__month=today.month,
+                                                    assignment__teaching_assistant__program="2", status="3").count()
+        phd_mhrd_pending = Feedback.objects.filter(date_submitted__month=today.month,
+                                                   assignment__teaching_assistant__program="2", status="1").count()
+
+        phd_vss_submitted = Feedback.objects.filter(date_submitted__month=today.month,
+                                                    assignment__teaching_assistant__program="3").count()
+        phd_vss_approved = Feedback.objects.filter(date_submitted__month=today.month,
+                                                   assignment__teaching_assistant__program="3", status="2").count()
+        phd_vss_rejected = Feedback.objects.filter(date_submitted__month=today.month,
+                                                   assignment__teaching_assistant__program="3", status="3").count()
+        phd_vss_pending = Feedback.objects.filter(date_submitted__month=today.month,
+                                                  assignment__teaching_assistant__program="3", status="1").count()
+
+        content = [
+            {
+                "programme": 'M.Tech',
+                "submitted": mtech_submitted,
+                "approved": mtech_approved,
+                "rejected": mtech_rejected,
+                "pending": mtech_pending,
+                "link": 'localhost:8000/pdf/1/'
```

Comment:
`localhost:8000/pdf/1/` hard coded links?

Reply:
I did this for the demo ...will change it when I get time ..